frontend/README.md

```diff
@@ -0,0 +1,118 @@
+## CustomerChurnWebapp
+
+Basic Web project with ExpressJS on NodeJS
+
+[![](https://img.shields.io/badge/IBM%20Cloud-powered-blue.svg)](https://bluemix.net)
+![Platform](https://img.shields.io/badge/platform-NODE-lightgrey.svg?style=flat)
+
+### Table of Contents
+* [Summary](#summary)
+* [Requirements](#requirements)
+* [Configuration](#configuration)
+* [Run](#run)
+* [Debug](#debug)
+
+<a name="summary"></a>
+### Summary
+The Web basic starter contains an opinionated set of files for web serving:
+
+- `index.html`
+- `results.html`
+- `404.html`
+- `500.html`
+
+
+
+<a name="enablement"></a>
+### IBM Cloud Enablement
+
+<a name="requirements"></a>
+### Requirements
+#### Local Development Tools Setup (optional)
+
+- Install the latest [NodeJS](https://nodejs.org/en/download/) 6+ LTS version.
+
+#### IBM Cloud development tools setup (optional)
+
+1. Install [IBM Cloud Developer Tools](https://console.bluemix.net/docs/cli/idt/setting_up_idt.html#add-cli) on your machine  
+2. Install the plugin with: `bx plugin install dev -r bluemix`
+
+
+#### IBM Cloud DevOps setup (optional)
+
+[![Create Toolchain](https://console.ng.bluemix.net/devops/graphics/create_toolchain_button.png)](https://console.ng.bluemix.net/devops/setup/deploy/)
+
+[IBM Cloud DevOps](https://www.ibm.com/cloud-computing/bluemix/devops) services provides toolchains as a set of tool integrations that support development, deployment, and operations tasks inside IBM Cloud. The "Create Toolchain" button creates a DevOps toolchain and acts as a single-click deploy to IBM Cloud including provisioning all required services. 
+
+***Note** you must publish your project to [Github](https://github.com/) for this to work.
+
+
+
+<a name="configuration"></a>
+### Configuration
+
+The project contains IBM Cloud specific files that are used to deploy the application as part of an IBM Cloud DevOps flow. The `.bluemix` directory contains files used to define the IBM Cloud toolchain and pipeline for your application. The `manifest.yml` file specifies the name of your application in IBM Cloud, the timeout value during deployment, and which services to bind to.
+
+Service credentials are taken from the VCAP_SERVICES environment variable if running IBM Cloud Cloud Foundry, from individual environment variables per service if running on IBM Cloud Container Service (see ./server/config/mappings.json), or from a config file if running locally, named`./server/config/localdev-config.js`.
```

Comment:
The file `./server/config/localdev-config.js` is missing.
There is only `server/config/local.json`
and this file contains only:
```
{
  "port": 3000
}
```
We will want this to run locally, as well as in the cloud, since many developers will want to test and debug on their laptop/workstation.
We do this in most patterns:
https://github.com/IBM/nlc-email-phishing#4-configure-credentials

See in that ^^ pattern how you can run in the IBM cloud, using vcap files, OR run locally, using a .env file for your credentials.
You can see how we use the `dotenv` module here:
https://github.com/IBM/nlc-email-phishing/blob/master/server/routes/classify.js#L19

Now the app can use either vcap or .env, depending on whether it runs locally or in the cloud, because they are available as environment variables:
https://github.com/IBM/nlc-email-phishing/blob/master/server/routes/classify.js#L28